### PR TITLE
Add property that manages if panels should be rerendered or not

### LINKED ISSUE #25322: [Bug]: Accordion re-renders panels instead of hidding/showing
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
System:
    OS: macOS 12.6
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 1.57 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Browsers:
    Chrome: 106.0.5249.119
    Firefox: 103.0.2
    Safari: 16.0
```

### Are you reporting Accessibility issue?

_No response_

### Reproduction

Add accordion with panels. When you click on a title panels show up and hide. But each time they rerenders instead of just hiding.

## Actual Behavior
Accordion panels are being rerendered each time. It causes issues because many requests are being done instead of one. You may check the actual behavior here in the [example](https://fluentsite.z22.web.core.windows.net/0.64.0/components/accordion/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#usage-accordion-panel-custom-title-example)

## Expected Behavior
Expect that the panels just hide or show without rerendering. The logic was available in v59 of the library. Possibly a flag needs to be added that controls the functionality. 
Here is an expected behavior [example] (https://fluentsite.z22.web.core.windows.net/0.59.0/components/accordion/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#usage-accordion-panel-custom-title-example)

### Logs

_No response_

### Requested priority

High

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by Hensorskyy**
## Current Behavior

Currently, there is no way to manage if panels have to be rendered each time or not

## New Behavior

Setting the property to true, panels will not be rendered each time

## Related Issue(s)

Fixes #25322

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/025e0740b1c47bd105b9cc6277072941bf855b8c'>025e0740b1c47bd105b9cc6277072941bf855b8c</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
SkeletonMinimalPerf.default | [408](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SkeletonMinimalPerf.default.html) | [362](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.13:1
TreeWith60ListItems.default | [197](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TreeWith60ListItems.default.html) | [175](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.13:1
AnimationMinimalPerf.default | [610](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AnimationMinimalPerf.default.html) | [560](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.09:1
AttachmentMinimalPerf.default | [180](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AttachmentMinimalPerf.default.html) | [167](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.08:1
TextMinimalPerf.default | [394](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TextMinimalPerf.default.html) | [366](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.08:1
ButtonMinimalPerf.default | [191](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonMinimalPerf.default.html) | [181](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.06:1
IconMinimalPerf.default | [768](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/IconMinimalPerf.default.html) | [724](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.06:1
VideoMinimalPerf.default | [844](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/VideoMinimalPerf.default.html) | [794](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.06:1
CarouselMinimalPerf.default | [518](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CarouselMinimalPerf.default.html) | [493](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.05:1
ChatMinimalPerf.default | [821](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatMinimalPerf.default.html) | [782](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.05:1
HeaderSlotsPerf.default | [898](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/HeaderSlotsPerf.default.html) | [856](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.05:1
MenuMinimalPerf.default | [960](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/MenuMinimalPerf.default.html) | [916](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.05:1
TreeMinimalPerf.default | [928](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TreeMinimalPerf.default.html) | [882](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.05:1
CardMinimalPerf.default | [620](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CardMinimalPerf.default.html) | [598](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.04:1
ListWith60ListItems.default | [712](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListWith60ListItems.default.html) | [683](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.04:1
TextAreaMinimalPerf.default | [560](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TextAreaMinimalPerf.default.html) | [541](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.04:1
ChatDuplicateMessagesPerf.default | [300](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [290](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.03:1
CheckboxMinimalPerf.default | [2330](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2253](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.03:1
FlexMinimalPerf.default | [315](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/FlexMinimalPerf.default.html) | [307](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.03:1
LayoutMinimalPerf.default | [401](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LayoutMinimalPerf.default.html) | [388](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.03:1
PortalMinimalPerf.default | [173](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/PortalMinimalPerf.default.html) | [168](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.03:1
RadioGroupMinimalPerf.default | [498](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [485](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [433](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ReactionMinimalPerf.default.html) | [420](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
BoxMinimalPerf.default | [370](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/BoxMinimalPerf.default.html) | [361](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.02:1
ButtonSlotsPerf.default | [613](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonSlotsPerf.default.html) | [603](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.02:1
ChatWithPopoverPerf.default | [416](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatWithPopoverPerf.default.html) | [406](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [854](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DialogMinimalPerf.default.html) | [840](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
EmbedMinimalPerf.default | [4088](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/EmbedMinimalPerf.default.html) | [4022](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1.02:1
ListCommonPerf.default | [733](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListCommonPerf.default.html) | [717](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListCommonPerf.default_base.html) | 1.02:1
ListMinimalPerf.default | [578](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListMinimalPerf.default.html) | [567](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.02:1
RosterPerf.default | [2364](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RosterPerf.default.html) | [2329](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RosterPerf.default_base.html) | 1.02:1
PopupMinimalPerf.default | [688](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/PopupMinimalPerf.default.html) | [674](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
ProviderMergeThemesPerf.default | [1317](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1297](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.02:1
ProviderMinimalPerf.default | [431](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ProviderMinimalPerf.default.html) | [423](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.02:1
SplitButtonMinimalPerf.default | [4885](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4811](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [765](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/StatusMinimalPerf.default.html) | [752](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
TableManyItemsPerf.default | [2201](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TableManyItemsPerf.default.html) | [2166](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.02:1
AttachmentSlotsPerf.default | [1221](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1210](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [1239](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/InputMinimalPerf.default.html) | [1232](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
ListNestedPerf.default | [632](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListNestedPerf.default.html) | [625](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ListNestedPerf.default_base.html) | 1.01:1
MenuButtonMinimalPerf.default | [1897](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1876](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.01:1
RefMinimalPerf.default | [236](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RefMinimalPerf.default.html) | [233](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.01:1
SegmentMinimalPerf.default | [387](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SegmentMinimalPerf.default.html) | [385](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.01:1
CustomToolbarPrototype.default | [2863](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CustomToolbarPrototype.default.html) | [2835](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [6333](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DatepickerMinimalPerf.default.html) | [6304](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
GridMinimalPerf.default | [376](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/GridMinimalPerf.default.html) | [376](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/GridMinimalPerf.default_base.html) | 1:1
HeaderMinimalPerf.default | [397](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/HeaderMinimalPerf.default.html) | [396](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1:1
ImageMinimalPerf.default | [454](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ImageMinimalPerf.default.html) | [454](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1:1
LoaderMinimalPerf.default | [717](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LoaderMinimalPerf.default.html) | [714](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1:1
SliderMinimalPerf.default | [1779](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SliderMinimalPerf.default.html) | [1773](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1:1
TableMinimalPerf.default | [458](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TableMinimalPerf.default.html) | [457](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TableMinimalPerf.default_base.html) | 1:1
DividerMinimalPerf.default | [393](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DividerMinimalPerf.default.html) | [397](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DividerMinimalPerf.default_base.html) | 0.99:1
DropdownMinimalPerf.default | [2874](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DropdownMinimalPerf.default.html) | [2893](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 0.99:1
ItemLayoutMinimalPerf.default | [1322](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1340](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 0.99:1
AccordionMinimalPerf.default | [161](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AccordionMinimalPerf.default.html) | [165](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.98:1
AlertMinimalPerf.default | [306](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AlertMinimalPerf.default.html) | [311](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.98:1
ButtonOverridesMissPerf.default | [1465](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1499](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.98:1
DropdownManyItemsPerf.default | [755](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DropdownManyItemsPerf.default.html) | [768](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 0.98:1
LabelMinimalPerf.default | [422](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LabelMinimalPerf.default.html) | [430](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.98:1
TooltipMinimalPerf.default | [2543](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TooltipMinimalPerf.default.html) | [2606](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.98:1
FormMinimalPerf.default | [416](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/FormMinimalPerf.default.html) | [429](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/FormMinimalPerf.default_base.html) | 0.97:1
ToolbarMinimalPerf.default | [1044](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ToolbarMinimalPerf.default.html) | [1075](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.97:1
AvatarMinimalPerf.default | [210](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AvatarMinimalPerf.default.html) | [222](https://fluentuipr.z22.web.core.windows.net/pull/25368/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 0.95:1

</details><div id="perfComment9424"/>

File: packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -93,6 +93,9 @@ export interface AccordionProps extends UIComponentProps, ChildrenComponentProps
    * Accessibility behavior if overridden by the user.
    */
   accessibility?: Accessibility<AccordionBehaviorProps>;
+
+  /** Manage if panels should be rerendered each time or just shown or hidden. */
+  disablePanelsRerendering?: boolean;
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx:98]:
```suggestion

  /** Manage if panels should be rendered always or based on their state. */
  alwaysRenderPanels?: boolean;
```

Please rename the prop as it's not about rerendering, it's what me initially confused 🙃

**Comment by layershifter** [comment]
Please also add an entry to our changelog in `packages/fluentui/CHANGELOG.md` 🙏

File: packages/fluentui/CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add new style to v0 Tooltip to match v9 Tooltip @GianoglioEnrico ([#24908](https://github.com/microsoft/fluentui/pull/24908))
```
Comment (fabricteam) [inline packages/fluentui/CHANGELOG.md:28]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25368&commitId=61a436c84c3a7be4ec6f2800ed8ca2b625deb2b7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Checked Mixed Teams.chromium.png|62| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Comment by Hensorskyy** [comment]
> 

@microsoft-github-policy-service agree [company="{Atlassian}"]

**Comment by Hensorskyy** [comment]
> @microsoft-github-policy-service agree [company="{Atlassian}"]

@microsoft-github-policy-service agree comany="Atlassian"

**Comment by Hensorskyy** [comment]
> @Hensorskyy the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree [company="{Atlassian

**Comment by Hensorskyy** [comment]
@microsoft-github-policy-service agree Atlassian

**Comment by layershifter** [comment]
@Hensorskyy thanks, set it to merge on CI pass.

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by layershifter** [comment]
@Hensorskyy it seems that CLA is not signed yet, can you please sign it?

**Comment by Hensorskyy** [comment]
@microsoft-github-policy-service agree

**Comment by Hensorskyy** [comment]
> @Hensorskyy please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by Hensorskyy** [comment]
@layershifter Should be good now

**Review by layershifter — APPROVED** [review]

**Pull request merged**